azext_iot/tests/iothub/conftest.py

```diff
@@ -693,6 +693,7 @@ def _cosmos_db_provisioner():
     collection_name = generate_hub_depenency_id()
     partition_key_path = "/test"
     location = "eastus"
+    print(f"--locations regionName={location}")
```

Comment:
Was this print meant to be left in the PR?

Reply:
lol nope